articles/azure-large-instances/work-with-ali-in-the-azure-portal.md

```diff
@@ -0,0 +1,271 @@
+This activity makes no changes on the ALI instance itself, but documents the changes to the unit's metadata in Azure.
+
+Another activity that gets recorded is adding a tag to or deleting a tag from an instance.
+
+## Add an Azure tag to or delete an Azure tag from an instance
+
+You can add Azure tags to an ALI instance or delete them using either the Portal or Azure CLI.  
+
+### [Portal](#tab/azure-portal)
+ 
+Tags get assigned just as they do when assigning tags to VMs.
+As with VMs, the tags exist in the Azure metadata.
+Tags have the same restrictions for ALI instances as for VMs.
+ 
+Deleting tags also works the same way as for VMs. 
+Both applying and deleting a tag is listed in the ALI instance's Activity log.
+
+### [Azure CLI](#tab/azure-cli)
+
+Assigning tags to ALI instances works the same as assigning tags for VMs.
+As with VMs, the tags exist in the Azure metadata.
+Tags have the same restrictions for ALI instances as for VMs.
+
+To add tags to an ALI instance, run the [az baremetalinstance update](/cli/azure/baremetalinstance#az-baremetalinstance-update) command:
+
+```azurecli
+az baremetalinstance update --resource-group DSM05a-T550 --instance-name orcllabdsm01 --set tags.Dept=Finance tags.Status=Normal
+```
+
+Use the same command to remove a tag:
+
+```azurecli
+az baremetalinstance update --resource-group DSM05a-T550 --instance-name orcllabdsm01 --remove tags.Dept
+```
+
+---
+
+### Check properties of an instance
+
+When you acquire the instances, you can go to the Properties section to view the data collected about the instances. 
+Data collected includes:
+
+* Azure connectivity
+* Storage backend
+* ExpressRoute circuit ID
+* Unique resource ID
+* Subscription ID
+
+This information is important in support requests and when setting up a storage snapshot configuration.
+
+Image TBD in next couple of weeks - IW 
+
+### Restart an ALI instance through the Azure portal
+
+There are various situations in which the operating system won't complete a restart, which requires a power restart of the ALI instance.
+
+You can do a power restart of the instance directly from the Azure portal or through Azure CLI.
+
+### [Portal](#tab/azure-portal)
+
+Select Restart and then Yes to confirm the restart. 
+
+ :::image type="content" source="../baremetal-infrastructure/media/connect-baremetal-infrastructure/baremetal-instance-restart.png" alt-text="Screenshot showing how to restart the ALI instance.":::
+
+When you restart an AKI instance, you'll experience a delay. 
+During this delay, the power state moves from **Starting** to **Started**, which means the OS has started up completely. 
+As a result, after a restart, you can only log into the unit once the state switches to **Started**. 
+
+### [Azure CLI](#tab/azure-cli)
+
+To restart an ALI instance, use the [az baremetalinstance restart](/cli/azure/baremetalinstance#az-baremetalinstance-restart) command:
+
+```azurecli
+az baremetalinstance restart --resource-group --instance-name 
```

Comment:
This line was corrected in wrong way... This line should be.
az baremetalinstance restart --resource-group MyResourceGroup --instance-name MyALIinstanceName

In addition to this line, please update all the commands in similar way.. replace hardcoded resource group name and instance name.

Reply:
@makotrik for the line 'Enable ER FastPath on the gateway connection.' that is above the H4 title "Declare Variable for ExpressRoute circuit" with cli commands. Is there a cli command to enable ER FastPath on the gateway connection?

Reply:
@vasthomsft the Connect your account, azurecli section, has been updated.